llvm/lib/CAS/OnDiskCAS.cpp

```diff
@@ -2061,6 +2064,17 @@ NamedTreeEntry OnDiskCAS::getInTree(const TreeProxy &Tree, size_t I) const {
   return makeTreeEntry(Record, I);
 }
 
+Expected<StringRef> OnDiskCAS::getNodeName(const NodeProxy &Node,
```

Comment:
This should not be API. Node doesn't have a name. This is just how object file format stores the node kind doesn't mean this is a generic function for Node.

If you want this function, you want a callback function in the visitor and implement this function in `llvm-cas-object-format` tool.